examples/DataStax/README.md

```diff
@@ -1,11 +1,15 @@
 # DataStax Node.js Driver usage samples
 
-This examples are from https://github.com/datastax/nodejs-driver/tree/master repository
+This examples are from the [datastax driver repository](https://github.com/datastax/nodejs-driver/tree/master).
 This folder contains examples on how to use some features of the DataStax Node.js Driver.
 
+They assume you have a running a ScyllaDB / Cassandra at the default port.
+You can provide ip address of the database by setting ``SCYLLA_URI`` env variable. Any port provided alongside ip will be ignored.
```

Comment:
Now I don't understand why we strip the port. It's part of configuration; Rust driver also expects `IP:PORT` format, but if PORT is not present, then default 9042 is used. IMO we should pass the port down the code, too.

Reply:
Previously, when I tested it, it didn't work if the port was provided. But I must have made a mistake, because it works now, so there is no need to remove the port.

Reply:
Previously, when I tested it, it didn't work if the port was provided. But I must have made a mistake, because it works now, so there is no need to remove the port.